docs/attributes-registry/scm.md

```diff
@@ -10,10 +10,10 @@
 
 This group defines the attributes for source control management (SCM) systems.
 
-| Attribute                       | Type   | Description                                                              | Examples                                                           | Stability                                                        |
-| ------------------------------- | ------ | ------------------------------------------------------------------------ | ------------------------------------------------------------------ | ---------------------------------------------------------------- |
-| `scm.repository.change.id`      | string | The id of the change (pull request / merge request) in the repository.   | `123`                                                              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `scm.repository.change.name`    | string | The name of the change (pull request / merge request) in the repository. | `pull-request-123`                                                 | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `scm.repository.name`           | string | The name of the repository.                                              | `opentelemetry-collector-contrib`                                  | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `scm.repository.ref.commit.sha` | string | The sha of the commit the ref (branch) is pointing to in the repository. | `9d59409acf479dfa0df1aa568182e43e43df8bbe28d60fcf2bc52e30068802cc` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `scm.repository.ref.name`       | string | The name of the ref (branch) in the repository.                          | `my-feature-branch`                                                | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Attribute                                  | Type   | Description                                                              | Examples                                                           | Stability                                                        |
+| ------------------------------------------ | ------ | ------------------------------------------------------------------------ | ------------------------------------------------------------------ | ---------------------------------------------------------------- |
+| `scm.repository.repository.change.id`      | string | The id of the change (pull request / merge request) in the repository.   | `123`                                                              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
We have `repository.repository` here.

Following the change `prefix: scm` → `prefix: scm.repository` in 2d557c66786b3219160bd46a5612bb60ec4329b8 should the attributes in `model/registry/scm.yaml` have the `repository.` prefix removed from the id?